static_data/aircraft_list.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
What is the difference between this and `original_aircraft_list.json`? They seem identical. If so, remove the redundant one